### PR TITLE
WIP - our development

### PR DESCRIPTION
**Pull request by schmunk42**
This is just for reviewing purposes, since we did a namespace change.
Please have a look and let us know if you'd basically merge these changes.

### CODE REVIEW

File: UsuarioLdapComponent.php [hunk 1]
```diff
@@ -314,28 +321,40 @@ public function events() {
...
 
             $username = $form->login;
             try {
+                Yii::debug('Searching LDAP user...', __METHOD__);
                 $ldapUser = $this->findLdapUser($username, 'cn');
+                Yii::debug(['Result for LDAP user', $ldapUser], __METHOD__);
             } catch (NoLdapUserException $e) {
                 $password = $form->password;
                 $user = User::findOne(['username' => $username]);
+                Yii::debug($user, __METHOD__);return;
```
Comment (maxxer) [inline UsuarioLdapComponent.php:350]: Is this `return` correct?

**Comment by maxxer** [comment]
Hi, thanks for the PR! Honestly, I'd prefer to keep the original namespace, as we were  the original authors. Do you think this can be a problem for you?

@mp1509 do you see anything which could stop this PR from being merged, other than the NS?

**Comment by schmunk42** [comment]
> Hi, thanks for the PR! Honestly, I'd prefer to keep the original namespace, as we were the original authors.

Yeah, sure ;)

>  Do you think this can be a problem for you?

Will figure out a way to revert this if needed.

**Comment by maxxer** [comment]
Please do, as said I'd prefer to retain the original NS.

Have you seen the comment on the `return` in the PR?

**Comment by schmunk42** [comment]
> Have you seen the comment on the return in the PR?

No?! In this PR?
I can't see anything here: https://github.com/YetOpen/yii2-usuario-ldap/pull/2/files

**Comment by maxxer** [comment]
I don't know how to link it ;)

immagine

**Comment by schmunk42** [comment]
@maxxer Good catch! I also think your review is still in "pending" state and therefore I can not see it.

**Review by maxxer — CHANGES_REQUESTED** [review]

**Comment by schmunk42** [comment]
@maxxer FYI - we are actively working on that, but we need to merge your latest changes and create a new PR

CC: @eluhr @H-Algabri

**Pull request merged**